dist/debian/debian_files_gen.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import string
+import os
+import shutil
+import re
+from pathlib import Path
+
+class DebianFilesTemplate(string.Template):
+    delimiter = '%'
+
+scriptdir = os.path.dirname(__file__)
+
+with open(os.path.join(scriptdir, 'changelog.template')) as f:
+    changelog_template = f.read()
+
+with open(os.path.join(scriptdir, 'control.template')) as f:
+    control_template = f.read()
+
+with open('build/SCYLLA-PRODUCT-FILE') as f:
+    product = f.read().strip()
+
+with open('build/SCYLLA-VERSION-FILE') as f:
+    version = f.read().strip().replace('.rc', '~rc').replace('_', '-')
```

Comment:
This line is not needed anymore, since our current version syntax is `x.x.x-rcX` , it should also removed from Scylla

Reply:
this file was copied as is from scylla-java-tools, I didn't change a thing here.

Reply:
as if we are changing this on core, we could change it across the sub-modules.

and if I've got you correctly, we shouldn't be replacing anything here ?

Reply:
yes, this condition is not relevant anymore. but since it's a new file. there is no point leaving it and then remove it IMO.

Reply:
removed this hack